spec/index.bs

```diff
@@ -1920,6 +1922,29 @@ interface GPUAdapterInfo {
         this value is not recommended. Applications which change their behavior based on the
         {{GPUAdapterInfo}}, such as applying workarounds for known driver issues, should rely on the
         other fields when possible.
+
+    : <dfn>properties</dfn>
```

Comment:
Is there a need/purpose for encapsulating these things in a new GPUAdapterProperties object?

Can the two new fields be included directly in GPUAdapterInfo?

Somewhat related in C-style APIs, I expect we would have a new chained struct, e.g. either GPUAdapterSubgroupInfo (or GPUAdapterPropertiesSubgroupInfo if we do introduce a new GPUAdapterProperties)

Reply:
I'd like input from @kainino0x and @toji. I don't have a strong opinion myself.

Reply:
+1, my recollection from the F2F discussion is that we'd put this directly inside of `GPUAdapterInfo`, not that we would nest `GPUAdapterProperties` inside of `GPUAdapterInfo`.

Reply:
We definitely agreed it goes inside GPUAdapterInfo (not on GPUAdapter or GPUDevice), don't think we really talked about whether it should be a sub-object or not. I don't have a strong feeling, but I don't think there's any need to use a sub-object.